pages/glossary/semantic-role.md

```diff
@@ -7,13 +7,13 @@ input_aspects:
   - DOM tree
 ---
 
-A _semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role)_ is a semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
+A _semantic association_, also called [role][], indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
 
-Valid semantic roles are defined by standards. For web content and applications these are defined within the various [WAI-ARIA specifications](https://act-rules.github.io/glossary/#wai-aria-specifications).
+Valid semantic associations are defined by standards. For web content and applications these are defined within the various [WAI-ARIA specifications][].
 
-The _semantic role_ of an element is determined by the first of these cases that applies:
+The _semantic role_ of an element is one way to give it a semantic association. It is determined by the first of these cases that applies:
```

Comment:
If it is "one way" what are the other ways?

Reply:
Both "explicit role" and "implicit role" are other ways (used in "semantic role" but nonetheless different.

We could also imagine a system with, say, the HTTP payload containing a table of xpath/semantic association in JSON format. That would be another way.